### DISCUSSION TITLE
How to disable "module._compile" verbose

### DISCUSSION THREAD

**Discussion by itxtoledo** [TypeStrong/ts-node#1416] [Official Recipes]
At some point ts-node-dev started showing these logs and I would like to disable it but I couldn't find anywhere explaining how

![image](https://user-images.githubusercontent.com/21026675/128019926-5c9848b2-50cc-4e0b-9551-fffd00e7294a.png)

**Comment by cspotcode** [comment]
TS_NODE_DEBUG env var
